attic/rfc_metrics.md

```diff
@@ -36,14 +51,55 @@ Internally, the function would look the following:
 
 Example:
 ```r
-fn = function(truth, response, positive, ...) {
-  assert_binary(truth, response = response, positive = positive)
-  fn_cm(cm(truth, response, positive))
-}
+MeasureFairness = R6Class("MeasureFairness", inherit = Measure, cloneable = FALSE,
```

Comment:
This is more a question to @mllg :
How do we ensure here that all metrics derived from this work irrespective of the task? 
Is "not checking" enough?